packages/pn-pa-webapp/src/App.tsx

```diff
@@ -38,26 +45,61 @@ const App = () => {
     (state: RootState) => state.userState.organizationParty
   );
   const { tos } = useAppSelector((state: RootState) => state.userState);
+  const currentStatus = useAppSelector((state: RootState) => state.appStatus.currentStatus);
 
   const dispatch = useAppDispatch();
+  const theme = useTheme();
   const { t, i18n } = useTranslation(['common', 'notifiche']);
   const { hasApiErrors } = useErrors();
 
   // TODO check if it can exist more than one role on user
   const role = loggedUser.organization?.roles[0];
   const idOrganization = loggedUser.organization?.id;
   const sessionToken = loggedUser.sessionToken;
+
   const menuItems = useMemo(() => {
+    const basicMenuItems: Array<SideMenuItem> = [
+      { label: t('menu.notifications'), icon: Email, route: routes.DASHBOARD },
+      /**
+       * Refers to PN-1741
+       * Commented out because beyond MVP scope
+       * 
+       * LINKED TO:
+       * - "<Route path={routes.API_KEYS}.../>" in packages/pn-pa-webapp/src/navigation/routes.tsx
+       * - BasicMenuItems in packages/pn-pa-webapp/src/utils/__TEST__/role.utilitytest.ts
+       */
+      // { label: menu.api-key, icon: VpnKey, route: routes.API_KEYS },
+      { 
+        label: t('menu.app-status'), 
+        // ATTENTION - a similar logic to choose the icon and its color is implemented in AppStatusBar (in pn-commons)
+        icon: () => currentStatus 
+          ? (currentStatus.appIsFullyOperative
+            ? <CheckCircleIcon sx={{ color: theme.palette.success.main }} />
```

Comment:
Check if we can avoid the use of useTheme(). When we use the material-ui components we can simply do this:
<Component  sx={{color: 'name of the color in the theme'}}/>

Reply:
Done, even when I personally prefer to use the theme, to avoid losing type discipline (`theme.palette.success.main` is checked by the editor and the compiler, while `"success.main"` is not).